Jenkinsfile

```diff
@@ -0,0 +1,8 @@
+standardPipeline {
+  standardNode {
+    checkoutStage {}
+    dockerBuildStage {
```

Comment:
Since we don't necessarily want to use this in prod, do we want to push this on artifactory?

Reply:
Prod does not have access to artifactory.
I would like to test it on a new cluster, maybe on ondemand but then I would like to run it in services for bit

Reply:
Services should be considered prod ;)